### DISCUSSION TITLE
Fit all play statistics (pp, UR, accuracy heatmap) on one screen

### DISCUSSION THREAD

**Discussion by Dimmitsaras** [ppy/osu#23984] [Ideas]
Statistics of a given play should all fit in a single screen without having to scroll

This way, people can share their play info with a single screenshot of the results page

 
Current Example; Performance breakdown or UR are cut off:
![osu_2023-06-21_11-35-37](https://github.com/ppy/osu/assets/35996393/311f8b45-7736-4963-98db-756281b14a3a)
![osu_2023-06-21_11-37-12](https://github.com/ppy/osu/assets/35996393/6df0b93e-0b7c-4708-ae24-34d9bc2feba0)

**Comment by peppy** [comment]
This is something I'm actively looking into improving!
